pkg/extensions/sync/sync_test.go

```diff
@@ -3680,7 +3680,7 @@ func TestSyncSignaturesDiff(t *testing.T) {
 		So(err, ShouldBeNil)
 
 		defer func() { _ = os.Chdir(cwd) }()
-		tdir, err := ioutil.TempDir("", "sigs")
+		tdir, err := os.MkdirTemp(", "sigs")
```

Comment:
Please fix this typo.

Also please remove ioutil from imports in this file.

Reply:
Done